method returning a forward defined type

The following program `sample.go` triggers an unexpected error:
```go
package main

import "fmt"  

var db dbWrapper

type dbWrapper struct {
    DB *cmap
}   

func (d *dbWrapper) get() *cmap {
    return d.DB
}
    
type cmap struct {
    name string
}

func (c *cmap) f() {
    fmt.Println("in f, c", c)
}

func f() {
    db.get().f()
}

func main() {
    db.DB = &cmap{name: "test"}
    db.get().f()
}
```

Expected result:
```console
$ go run ./sample.go
test
```

Got:
```console
$ yaegi ./sample.go
./sample.go:26:2: undefined selector: f
```

--- a/_test/method31.go
+++ b/_test/method31.go
@@ -1,0 +1,29 @@
+package main
+
+import "fmt"
+
+var db dbWrapper
+
+type dbWrapper struct {
+	DB *cmap
+}
+
+func (d *dbWrapper) get() *cmap {
+	return d.DB
+}
+
+type cmap struct {
+	name string
+}
+
+func (c *cmap) f() {
+	fmt.Println("in f, c", c)
+}
+
+func main() {
+	db.DB = &cmap{name: "test"}
+	db.get().f()
+}
+
+// Output:
+// in f, c &{test}

--- a/interp/gta.go
+++ b/interp/gta.go
@@ -125,6 +125,9 @@ func (interp *Interpreter) gta(root *node, rpath, pkgID string) ([]*node, error)
 				// Add a function symbol in the package name space
 				sc.sym[n.child[1].ident] = &symbol{kind: funcSym, typ: n.typ, node: n, index: -1}
 			}
+			if n.typ.incomplete {
+				revisit = append(revisit, n)
+			}
 			return false
 
 		case importSpec:

